Makefile

```diff
@@ -1,5 +1,6 @@
 .SECONDEXPANSION:
 
+BEST:=$(shell if which ocamlopt > /dev/null; then echo native; else echo byte; fi)
```

Comment:
Could the `which` be dropped here, like this:

`
BEST:=$(shell if ocamlopt > /dev/null; then echo native; else echo byte; fi)
`
?

Some systems, such as NixOS, apparently [don't include `which` by default](https://github.com/ocaml/opam-repository/pull/5649#issuecomment-194786216), and I'd rather avoid adding another [opam dependency](https://opam.ocaml.org/packages/conf-which/conf-which.1/).